tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -94,15 +89,15 @@ def test_graph_mode(self):
                     self.assertTrue(tf.reduce_all(eager_outputs[key] == compiled_outputs[key]))
 
     @slow
-    def test_saved_model(self):
+    def test_export_for_inference(self):
         for tf_tokenizer in self.tf_tokenizers:
             model = ModelToSave(tokenizer=tf_tokenizer)
             test_inputs = tf.convert_to_tensor(self.test_sentences)
             out = model(test_inputs)  # Build model with some sample inputs
             with TemporaryDirectory() as tempdir:
                 save_path = Path(tempdir) / "saved.model"
-                model.save(save_path)
-                loaded_model = tf.keras.models.load_model(save_path)
-            loaded_output = loaded_model(test_inputs)
+                model.export(save_path)
```

Comment:
This is the recommended TF/Keras way to export models for inference now, and the old approach was always a bit shaky.